packages/fast-form-generator-react/src/form/form-category.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import IFormItemCommon from "./form-item";
+import styles from "./form-category.style";
+import { IFormCategoryClassNameContract } from "../class-name-contracts/";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * Select state interface
+ */
+export interface IFormCategoryProps {
+
+    /**
+     * Passes the category item
+     */
+    categoryItem: JSX.Element[];
+
+    /**
+     * Passes the category title
+     */
+    title: string;
+
+    /**
+     * Is the category expandable
+     */
+    Expandable?: boolean;
+
+    /**
+     * Is the category expanded
+     */
+    isExpanded?: boolean;
+
+    /**
+     * Passes the id
+     */
+    id?: string;
+}
+
+export interface IFormCategoryState {
+    isCategoryExpanded: boolean;
+}
+
+/**
+ * Schema form component definition
+ * @extends React.Component
+ */
+/* tslint:disable-next-line */
+class FormCategory extends React.Component<IFormCategoryProps & IManagedClasses<IFormCategoryClassNameContract>, IFormCategoryState> {
+
+    constructor(props: IFormCategoryProps & IManagedClasses<IFormCategoryClassNameContract>) {
+        super(props);
+
+        if (this.props.Expandable) {
+            this.state = {

```

Comment:
You probably want to set the state regardless so that it has the expected values (and is not undefined) - you can change to `this.state = { isCategoryExpanded: !!this.props.isExpanded };` to do this and remove the if statement. The `!!` return a boolean even if `this.props.isExpanded` is undefined.